client/src/components/DisplayBookings/CardImage/CardImage.tsx

```diff
@@ -0,0 +1,17 @@
+import clsx from 'clsx';
+import useStyles from './useStyle';
+
+interface Props {
+  isUpcoming?: boolean;
+}
+
+export default function CardImage({ isUpcoming }: Props): JSX.Element {
```

Comment:
Can we please delete this component as the HTML element within this component is already a part of `BookingCard`?